srcpkgs/Aegisub/template

```diff
@@ -1,61 +1,42 @@
 # Template file for 'Aegisub'
 pkgname=Aegisub
-version=3.2.2
-revision=13
-build_style=gnu-configure
-configure_args="--disable-update-checker --with-alsa --with-ffms2
- --with-wx-config=wx-config-gtk3 $(vopt_with fftw fftw3) $(vopt_with openal)
- $(vopt_with portaudio)"
-hostmakedepends="automake gettext-devel intltool libtool LuaJIT pkg-config"
+version=3.3.2
+revision=1
+_githash=723d01d13088d9b1bccc821f1129d045a0b74e2c
+build_style=cmake
+build_helper=cmake-wxWidgets-gtk3
+cmake_builddir="BUILD"
+configure_args="-DwxWidgets_CONFIG_EXECUTABLE=$WX_CONFIG
+ -DWITH_PORTAUDIO=$(vopt_if portaudio ON OFF)
+ -DWITH_OPENAL=$(vopt_if openal ON OFF) -DWITH_FFTW3=$(vopt_if fftw ON OFF)"
+hostmakedepends="gettext-devel intltool pkg-config git"
 makedepends="alsa-lib-devel boost-devel libcurl-devel fontconfig-devel
- freetype-devel hunspell-devel icu-devel libass-devel libffms2-devel LuaJIT-devel
+ freetype-devel hunspell-devel icu-devel libass-devel libffms2-devel
  MesaLib-devel wxWidgets-gtk3-devel $(vopt_if fftw fftw-devel)
  $(vopt_if openal libopenal-devel) $(vopt_if portaudio portaudio-devel)"
-short_desc="Tool for creating and modifying subtitles"
-maintainer="Enno Boland <gottox@voidlinux.org>"
+short_desc="Subtitle editor (ssa, ass, srt)"
+maintainer="pudiva <rogi@skylittlesystem.org>"
 license="BSD-3-Clause, MIT"
-homepage="http://www.aegisub.org"
-distfiles="https://github.com/Aegisub/Aegisub/archive/v${version}.tar.gz"
-checksum=62757dd491455268a240f983b59734a801cc2e899039a7493deeaf5e24a61dcd
+homepage="https://github.com/wangqr/Aegisub"
+nocross=yes
 
 build_options="fftw openal portaudio"
 build_options_default="fftw"
 
 desc_option_fftw="Enable support for rendering of audio waveforms/spectrum"
 desc_option_openal="Enable support for openal"
 
-LDFLAGS+=" -pthread"
-
-if [ "$XBPS_TARGET_NO_ATOMIC8" ]; then
-	makedepends+="libatomic-devel"
-	configure_args+=" --with-libatomic"
-fi
-
-if [ "$CROSS_BUILD" ]; then
-	# When cross building the detection of iconv const parameters is wrong
-	configure_args+=" agi_cv_with_iconv_const=no"
-	# and cross is broken because of boost::icu_regex_trais anyway ...
-	broken="/usr/aarch64-linux-gnu/usr/include/boost/regex/v4/perl_matcher_common.hpp:512: undefined reference to 'boost ::icu_regex_traits::isctype(int, unsigned long) const'"
-fi
-
-case "$XBPS_TARGET_MACHINE" in
-	*musl) broken="segfaults on startup";;
-esac
-
-post_extract() {
-	# Fix shebang to use system luajit instead of vendored minilua
-	sed -e \
-		"s:../vendor/luajit/src/host/minilua:/usr/bin/luajit:" \
-		-i tools/respack.lua
+do_fetch() {
+	git clone git://github.com/wangqr/Aegisub.git $wrksrc
+	cd $wrksrc
+	git checkout $_githash
 }
 
 pre_configure() {
 	# Aegisub fails to start on Wayland, so we have to force GDK to start it
 	# with the x11 backend
 	sed "s/Exec=@AEGISUB_COMMAND@ %f/Exec=env GDK_BACKEND=x11 @AEGISUB_COMMAND@ %f/" -i \
 		packages/desktop/aegisub.desktop.template.in
```

Comment:
I think you forgot to remove this code block?

Reply:
I re-added it as @paper42 said it was still broken on wayland https://github.com/void-linux/void-packages/pull/32162#pullrequestreview-737696497

Reply:
Sure, but the file in https://github.com/wangqr/Aegisub/blob/dev/packages/desktop/aegisub.desktop.template.in already  applies the variable (if you switch to using `vsed` there it will print a warning that it didn't replace anything).

Reply:
ohhh sorry my confusion

Reply:
no worries ;)